docs/glossary.rst

```diff
@@ -0,0 +1,19 @@
+.. _kinto-glossary:
+
+Glossary
+########
+
+
+.. glossary::
+
+    Cliquet Protocol
+            A system of rules that explains the way to interact with Kinto
+            :term:`endpoints`, and how Kinto data is organized.
```

Comment:
It's already in the Cliquet glossary. Ideally we should have a unified glossary across the two docs. I guess we can import it like we do for other cliquet docs